nixos/modules/system/boot/loader/limine/limine.nix

```diff
@@ -0,0 +1,271 @@
+      description = ''
+        Whether or not to install the limine EFI files.
+      '';
+    };
+
+    efiInstallAsRemovable = lib.mkEnableOption null // {
+      default = true;
+      description = ''
+        Whether or not to install the limine EFI files as removable.
+
+        See {option}`boot.loader.grub.efiInstallAsRemovable`
+      '';
+    };
+
+    biosSupport = lib.mkEnableOption null // {
+      default = !cfg.efiSupport && pkgs.stdenv.hostPlatform.isx86;
+      defaultText = lib.literalExpression "!boot.loader.limine.efiSupport && pkgs.stdenv.hostPlatform.isx86";
+      description = ''
+        Whether or not to install limine for BIOS.
+      '';
+    };
+
+    biosDevice = lib.mkOption {
+      default = "nodev";
+      type = lib.types.str;
+      description = ''
+        Device to install the BIOS version of limine on.
+      '';
+    };
+
+    partitionIndex = lib.mkOption {
+      default = null;
+      type = lib.types.nullOr lib.types.int;
+      description = ''
+        The 1-based index of the dedicated partition for limine's second stage.
+      '';
+    };
+
+    enrollConfig = lib.mkEnableOption null // {
+      default = cfg.panicOnChecksumMismatch;
+      defaultText = lib.literalExpression "boot.loader.limine.panicOnChecksumMismatch";
+      description = ''
+        Whether or not to enroll the config.
+        Only works on EFI!
+      '';
+    };
+
+    forceMbr = lib.mkEnableOption null // {
+      description = ''
+        Force MBR detection to work even if the safety checks fail, use absolutely only if necessary!
+      '';
+    };
+
+    style = {
+      wallpapers = lib.mkOption {
+        default = [ ];
+        type = lib.types.listOf lib.types.path;
+        description = ''
+          A list of wallpapers.
+          If more than one is specified, a random one will be selected at boot.
+        '';
+      };
+
+      wallpaperStyle = lib.mkOption {
+        default = "streched";
+        type = lib.types.enum [
+          "centered"
+          "streched"
+          "tiled"
+        ];
+        description = ''
+          How the wallpaper should be fit to the screen.
+        '';
+      };
+
+      backdrop = lib.mkOption {
+        default = null;
+        type = lib.types.nullOr lib.types.str;
+        description = ''
+          Color to fill the rest of the screen with when wallpaper_style is centered in RRGGBB format.
+        '';
+      };
+
+      interface = {
+        resolution = lib.mkOption {
+          default = null;
+          type = lib.types.nullOr lib.types.str;
+          description = ''
+            The resolution of the interface.
+          '';
+        };
+
+        branding = lib.mkOption {
+          default = null;
+          type = lib.types.nullOr lib.types.str;
+          description = ''
+            The title at the top of the screen.
+          '';
+        };
+
+        brandingColor = lib.mkOption {
+          default = null;
```

Comment:
```suggestion
          default = null;
          example = "64CAFE";
```

Reply:
I wrote the description wrong it's not in RRGGBB format.

Reply:
Ah sorry